### ISSUE TITLE
ValueError: Invalid partition object.

### ISSUE THREAD

**Issue by abegong** [fivetran/great_expectations#145]
It looks like `None`s are included in the denominator when creating categorical partitions. This causes `expect_column_chisquare_test_p_value_greater_than` to raise an `Invalid partition object` error.

```
df = ge.dataset.PandasDataSet({
    'education' : ["A", "B", "A", "B", None]
})
my_part = ge.dataset.util.categorical_partition_data(df["education"])
print "Weights :", my_part["weights"]
print "Sum(Weights) :", sum(my_part["weights"])

df.expect_column_chisquare_test_p_value_greater_than(column_name, my_part)
```

```
Weights : [ 0.4  0.4]
Sum(Weights) : 0.8
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-61-6073c7d6718e> in <module>()
      6 print "Sum(Weights) :", sum(my_part["weights"])
      7 
----> 8 df.expect_column_chisquare_test_p_value_greater_than(column_name, my_part)

/Users/abe/anaconda2/lib/python2.7/site-packages/great_expectations/dataset/util.pyc in f(*args, **kwargs)
     68         @wraps(self.mthd, assigned=('__name__', '__module__'))
     69         def f(*args, **kwargs):
---> 70             return self.mthd(obj, *args, **kwargs)
     71 
     72         return self.use_parent_doc(f, overridden)

/Users/abe/anaconda2/lib/python2.7/site-packages/great_expectations/dataset/base.pyc in wrapper(self, *args, **kwargs)
    116                             return_obj = False
    117                     else:
--> 118                         raise(err)
    119 
    120                 #Add a "success" object to the config

ValueError: Invalid partition object.
```

**Comment by abegong** [comment]
This might be fixed by PR #135 .

**Comment by jpcampb2** [comment]
This issue was not fixed in #135, so will need a separate fix. Your diagnosis is correct: the problem is that the numerator is ignoring NA and the denominator is not.

Because the default behavior of all aggregate expectations is to drop NA values, I think it is right to adjust the denominator to also drop NA; I'll make a quick PR to fix this.
